### PR TITLE
bpo-40465: Document random module changes in 3.11 What's new

### PR DESCRIPTION
**Pull request by hrnciar**
https://bugs.python.org/issue40465

### CODE REVIEW

File: Doc/whatsnew/3.11.rst [hunk 1]
```diff
@@ -287,6 +287,15 @@ os
   (Contributed by Dong-hee Na in :issue:`44611`.)
 
 
+random
+------
+
+* The *population* parameter of :func: `random.sample` must be a sequence.
```
Comment (hugovk) [inline Doc/whatsnew/3.11.rst]: This should fix the docs build:

```suggestion
* The *population* parameter of :func:`random.sample` must be a sequence.
```

**Review by hugovk — CHANGES_REQUESTED** [review]

**Review by serhiy-storchaka — DISMISSED** [review]
It is not a correct place for this note because it is not a new feature. From user's point of view it is a breaking change which can break a code that worked before (even if it issued deprecation warnings). It should be documented in sections "Porting to Python 3.11", subsection "Changes in the Python API" or in section "Removed" (different changes may be in different sections).

**Comment by vstinner** [comment]
@serhiy-storchaka:
> It should be documented in sections "Porting to Python 3.11"

Done: @hrnciar moved the What's New entry to Porting to Python 3.11 > Changes in the Python API.

**Review by serhiy-storchaka — APPROVED** [review]

**Review by vstinner — COMMENTED** [review]
Post-merge review (@serhiy-storchaka was faster than me :-)): LGTM! This change properly describes https://bugs.python.org/issue40465 changes.

**Comment by serhiy-storchaka** [comment]
BTW it was first time I used GitHub client for Android.

**Pull request merged**